src/renderer_gl.cpp

```diff
@@ -6991,6 +7030,44 @@ namespace bgfx { namespace gl
 					, height
 					, bx::uint32_imax(depth, 1)
 					) );
+				}
+		}
+		else
```

Comment:
`else if (BX_ENABLED(BGFX_GL_BLIT_EMULATION) )`

Reply:
And then remove all FBO caching. It's slow anyway...

Reply:
Done !